pai/job/_training_job.py

```diff
@@ -709,8 +709,7 @@ def _training_job_base_output(self, job_name):
             f"{to_plain_text(job_name)}_{random_str(6)}",
         )
         base_output_path = (
-            f"oss://{bucket_name}.{session.oss_endpoint}/"
-            f"{storage_path}"
+            f"oss://{bucket_name}.{session.oss_endpoint}/" f"{storage_path}"
```

Comment:
auto fix? looks strange

Reply:
The previous commit resulted in a failure of the lint test. Therefore, we should address the lint issues first.